components/inputs/input-color.js

```diff
@@ -277,8 +276,6 @@ class InputColor extends FocusMixin(FormElementMixin(LocalizeCoreElement(LitElem
 
 		super.updated(changedProperties);
 
-		if (changedProperties.has('label') || changedProperties.has('type')) this._validateLabel();
```

Comment:
This use case forced me to add support for "dependent properties" -- essentially other properties to watch and re-validate when they change.